src/alchemlyb/parsing/gmx.py

```diff
@@ -243,6 +243,36 @@ def extract_dHdl(xvg, T, filter=True):
     return dHdl
 
 
+def extract(xvg, T, filter=True):
```

Comment:
Add the decorator

Reply:
The decorator shouldn't be needed, as the `extract` function just returns a `dict` that is built calling the original `extract_*` functions, which have the correct decorator applied to them.
The only `extract` function that needs the new decorator is the AMBER one, as here it's the other way around, it's the `extract_dHdl/extract_u_nk` that calls the `extract` function.